web_output_block.py

```diff
@@ -1,27 +1,28 @@
 import json
-from nio.common.block.base import Block
-from nio.common.discovery import Discoverable, DiscoverableType
-from nio.metadata.properties import VersionProperty, ExpressionProperty, \
+from nio.block.base import Block
+from nio.util.discovery import discoverable
+from nio.properties import VersionProperty, Property, \
     PropertyHolder, ListProperty
 from .broker import RequestResponseBroker
 
 
 class ResponseHeader(PropertyHolder):
-    header_name = ExpressionProperty(title='Name', default='Content-Type')
-    header_val = ExpressionProperty(title='Value', default='application/json')
+    header_name = Property(title='Name', default='Content-Type')
+    header_val = Property(title='Value', default='application/json')
 
 
-@Discoverable(DiscoverableType.block)
+@discoverable
 class WebOutput(Block):
 
     version = VersionProperty('1.0.0')
 
-    id_val = ExpressionProperty(title='Request ID', default='{{ $id }}')
+    id_val = Property(title='Request ID', default='{{ $id }}')
 
-    response_out = ExpressionProperty(title='Response Body', default='')
-    response_status = ExpressionProperty(
+    response_out = Property(title='Response Body', default='')
+    response_status = Property(
```

Comment:
It could be, especially considering the code does `rsp_status = int(self.response_status(sig))`. It was previously an expression property so my upgrade script turns that into just `Property`. The only downside to making it an int property would be that it's difficult to set those as an expression with the current builder.